test/functional/kaui/subscriptions_controller_test.rb

```diff
@@ -22,7 +22,8 @@ class Kaui::SubscriptionsControllerTest < Kaui::FunctionalTestHelper
         :bundle_id => @bundle.bundle_id,
         :account_id => @account.account_id,
         :product_category => 'ADD_ON'
-    assert assigns(:plans).size > 0, 'Plans were not created'
+    assert_redirected_to account_bundles_path(@bundle.subscriptions.first.account_id)
+    assert_equal 'No available add-on for product Sports', flash[:error]
```

Comment:
I think the expectation was correct: the tests need to run with the `SpyCarAdvanced.xml` catalog (loaded when using the [killbill-server.properties](https://github.com/killbill/killbill/blob/2e8156f27ee9368b1d7b815a440af5829f9000aa/profiles/killbill/src/main/resources/killbill-server.properties#L32) config file).

This is an old limitation when we didn't support dynamic catalogs. Eventually, we should make sure these tests simply upload whatever catalog they need instead, similar to our [integration tests](https://github.com/killbill/killbill-integration-tests/blob/8a3299dd41115dd1abffb044a0d78901b3b1a882/killbill-integration-tests/core/test_catalog.rb#L11) (not suggesting you make the change at this point, but it's the direction we should be headed).